pkg/app/piped/cloudprovider/cloudrun/cloudrun.go

```diff
@@ -38,8 +38,8 @@ var (
 type Service run.Service
 
 type Client interface {
-	Apply(ctx context.Context, sm ServiceManifest) (*Service, error)
-	List(ctx context.Context) error
```

Comment:
oh, don't even know we did not use this 👀

Reply:
Yeah, it was expected to be used in the drift detector package. But currently, that feature is not implemented yet.